helix-view/src/editor.rs

```diff
@@ -114,7 +127,7 @@ impl Editor {
         }
     }
 
-    pub fn switch(&mut self, id: DocumentId, action: Action) {
+    pub fn switch(&mut self, id: DocumentId, action: Action, pos: Option<(u32, u32)>) {
```

Comment:
I don't like the extra parameter, why not just make it `switch` + `jump_to` if there's a position? That's how the goto jumps work across files.

Reply:
Fixed